node-graph/graph-craft/src/proto.rs

```diff
@@ -370,7 +370,8 @@ impl ProtoNetwork {
 				self.replace_node_id(&outwards_edges, node_id, compose_node_id, true);
 			}
 		}
-		self.reorder_ids();
+		self.reorder_ids()?;
+		return Ok(());
```

Comment:
You should be able to just write `Ok(())` here instead of adding an explicit return right? Btw. did you run `cargo clippy` on the code?

Reply:
Yup, and I've just run `cargo clippy` - nothing else seems to be showing up. This return statement has now been removed!